lib/evmone/tracing.cpp

```diff
@@ -25,21 +26,18 @@ class HistogramTracer : public Tracer
     {
         const int32_t depth;
         const uint8_t* const code;
-        const char* const* const opcode_names;
         uint32_t counts[256]{};
 
-        Context(int32_t _depth, const uint8_t* _code, const char* const* _opcode_names) noexcept
-          : depth{_depth}, code{_code}, opcode_names{_opcode_names}
-        {}
+        Context(int32_t _depth, const uint8_t* _code) noexcept : depth{_depth}, code{_code} {}
     };
 
     std::stack<Context> m_contexts;
     std::ostream& m_out;
 
     void on_execution_start(
-        evmc_revision rev, const evmc_message& msg, bytes_view code) noexcept override
+        evmc_revision /*rev*/, const evmc_message& msg, bytes_view code) noexcept override
     {
-        m_contexts.emplace(msg.depth, code.data(), evmc_get_instruction_names_table(rev));
+        m_contexts.emplace(msg.depth, code.data());
```

Comment:
In previous version we took only instructions from current revision. In the new one we take all instructions from all revisions. Not sure if it's ok.

Reply:
Yes, this is right. We lost some precision, e.g. some not-yet-defined instructions may be printed by name. I think this is fine for now. Later we can create tables with revision dimension (similar to EVMC). There is TODO added for this.

Reply:
Ok. Fine for me.